docs/content/v2024.2/yugabyte-platform/back-up-restore-universes/disaster-recovery/_index.md

```diff
@@ -64,7 +64,7 @@ Blog: [Using YugabyteDB xCluster DR for PostgreSQL Disaster Recovery in Azure](h
 
```

Comment:
Why is this under the Backup section?

Reply:
What are you referring to? Disaster Recovery generally? It could go under Manage.

Reply:
Yes Disaster Recovery generally